src/commands/Misc/reddituser.ts

```diff
@@ -81,31 +79,27 @@ export class UserPaginatedMessageCommand extends PaginatedMessageCommand {
 				embed
 					.addField(
 						`__${titles.bestComment}__`,
```

Comment:
Why not use `underscore(titles.bestComment)`?

Reply:
Forgot lol